server/db/postgres.js

```diff
@@ -34,6 +34,13 @@ var postgres = {
       Object.keys(row).forEach(function(k) {
         output[camelize(k)] = row[k];
       });
+      // special case: json extracted_data blob
+      if (output.extractedData) {
+        Object.keys(output.extractedData).forEach(function(key) {
+          output.extractedData[camelize(key)] = output.extractedData[key];
+          delete output.extractedData[key];
```

Comment:
derp. if the key is 'images' and doesn't contain underscores, this will delete it